node/src/blockchain/blockchain_interface.rs

```diff
@@ -904,31 +986,83 @@ mod tests {
         assert_eq!(token_balance, U256::from(1))
     }
 
-    //test with real web3
     #[test]
     fn blockchain_interface_non_clandestine_can_transfer_tokens() {
+        init_test_logging();
         let mut transport = TestTransport::default();
         transport.add_response(json!(
-            "0x0000000000000000000000000000000000000000000000000000000000000001"
+            "0xe26f2f487f5dd06c38860d410cdcede0d6e860dab2c971c7d518928c17034c8f"
         ));
+        let (recorder, _, recording_arc) = make_recorder();
```

Comment:
Could you name `recorder` and `recording_arc` so that it's clear what they're recording? For example, you could call them `accountant` and `accountant_recording_arc`, unless you're philosophically opposed to the BlockchainInterface knowing about the Accountant, which is reasonable; but still, something more intention-revealing than "recorder" would help the readability.